.github/workflows/reusable-performance.yml

```diff
@@ -298,6 +297,9 @@ jobs:
       - name: Compare results
         run: node ./tests/performance/compare-results.js ${{ runner.temp }}/summary.md
 
+      - name: Check date time
+        run: echo "COMMITTED_AT_TEST=$(git show -s $GITHUB_SHA --format='%cI')" >> $GITHUB_ENV
```

Comment:
After https://github.com/WordPress/wordpress-develop/pull/8072/commits/2b7099cd7270fb1793d06f782329dec388a04371 we get the Date time. 

See
<img width="734" alt="Screenshot 2025-01-02 at 12 34 33 PM" src="https://github.com/user-attachments/assets/7e452057-8f79-4ab1-a8f0-fa7cb5b09483" />